testing/conftest.py

```diff
@@ -136,6 +136,23 @@ def pytester(pytester: Pytester, monkeypatch: MonkeyPatch) -> Pytester:
     return pytester
 
 
+class _ColorMapping:
```

Comment:
You could declare `COLORS: Dict[str, str]` here, so that mypy knows that it exists.